review

```diff
@@ -125,7 +92,7 @@ review() {
     # Create rev branch:
     # - new one if there is no rev branch yet for this feature
     # - with an increased number if there is already some rev branches for this feature
-    coreBranchName="${currentBranch}_${SUFFIX}"
+    coreBranchName="${currentBranch}_${REVIEW_SUFFIX}"
```

Comment:
I cannot see where REVIEW_SUFFIX comes from?

Reply:
It's declared in touk-bash-gerrit – I also need it in cleanup, so I extracted it.
Perhaps I should adapt the `obtain` strategy? 🤔